Archive C23_WMDE_Desktop_DE_23 and C23_WMDE_Desktop_DE_21

Sperling-0 opened the pull request:
Archive C23_WMDE_Desktop_DE_21 and C23_WMDE_Desktop_DE_23

- Based on the VAR of test 21
- VAR has an early banner
- 23 does not show up as commits in `banners/desktop` because we merged 24 too early (original commit id of 23 branch: https://github.com/wmde/fundraising-banners/commit/34dafdb094bd24da6d47bbd88a984240fafc8159)

Ticket: https://phabricator.wikimedia.org/T353631
Co-authored-by: <abban.dunne@wikimedia.de>

Abban commented:
Can I remove any of the other folders?

moiikana commented:
wikipedia.de_OLD can be removed as well, it was just an old clone to make development easier

Abban committed:
Archive C23_WMDE_Desktop_DE_21

Ticket: https://phabricator.wikimedia.org/T353631

Sperling-0 committed:
Archive C23_WMDE_Desktop_DE_23

- Based on the VAR of test 21
- VAR has an early banner
- 23 does not show up as commits in `banners/desktop` because we merged 24 too early (original commit id of 23 branch: 34dafdb094bd24da6d47bbd88a984240fafc8159)

Ticket: https://phabricator.wikimedia.org/T353631
Co-authored-by: <abban.dunne@wikimedia.de>
